polkadot/node/collation-generation/src/lib.rs

```diff
@@ -602,10 +585,10 @@ async fn construct_and_distribute_receipt(
 	let ccr = CandidateReceipt {
 		commitments_hash: commitments.hash(),
 		descriptor: CandidateDescriptor {
-			signature: key.sign(&signature_payload),
+			signature: CollatorSignature::from_slice(&vec![0u8; 64]).expect("64 bytes; qed"),
```

Comment:
Actually, using a dummy signature should only be possible after validators have upgraded, otherwise their collations will be rejected by validators still checking these fields.